audio/ddptools/Portfile

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+name            ddptools
+version         1.1
+revision        0
+
+categories      audio
+platforms       {darwin any}
+maintainers     {breun.nl:nils @breun} openmaintainer
+license         none
+
+description     DDP Mastering Tools
+
+long_description The DDP Mastering Tools are a small set of command line \
+                programs which read and write DDP images as used for Red \
+                Book CD audio mastering. They let you\
+                \n\
+                \n* create DDP 2.0 masters from a cue/wav images,\
+                \n* inspect DDP masters by displaying the full meta-data found \
+                in those masters, a feature rarely found in traditional DDP \
+                software,\
+                \n* convert DDP masters to cue/wav for easier playback, or \
+                burning of CD reference copies.
+
+homepage        http://ddp.andreasruge.de
+master_sites    http://ddp.andreasruge.de/dist/
+
+distname        ${name}-${version}-osx
+
+checksums       rmd160  2541cb617c437ee3d61499504d370d5dbb015348 \
+                sha256  dc2a264b591c50c72862091de464d1d5b1740909035cb1ee8d513213ab502e0a \
+                size    111753
+
+worksrcdir      ${name}-${version}
+
+use_configure   no
+
+build {}
```

Comment:
Is the source code available? We really prefer not to just install binaries...

Reply:
No, sadly the source is not available. http://ddp.andreasruge.de/ says: “The DDP Mastering Tools are available as binaries for Linux, OSX and Windows and can be used free of charge. Due to DCA’s licensing conditions they are not released under an open source license at the moment.”

Reply:
I am not sure whether that model fits for MaxPorts; there is no real benefit the. People can just download it from upstreams' website. Since the last release was in 2018 updates are unlikely...

Reply:
I do see the benefit. I avoid installing CLI software outside a package manager as much as possible, because it’s so much easier to track what is installed on a system when using a package manager, and it’s so much easier to have a uniform single command to install something instead of having to look up the installation instructions for every single piece of software every time you want to install it on a new system. This is the main reason for me to use MacPorts.

DDP is a stable format for audio CD mastering projects, so indeed frequent updates are not expected, but I’m volunteering to maintain this port.

Reply:
I'm not 100% pleased about no-source packages either, but I understand the desire to have third party software managed on one's box, and I think MacPorts exists to make the lives of the people in the community easier, not to satisfy my aesthetic sense. I think we can merge this now.